VideoPlayer.js

```diff
@@ -682,6 +682,12 @@ export default class VideoPlayer extends Component {
             })
         }
 
+    	if (this.state.rate !== nextProps.rate) {
+      	   this.setState({
+        	rate: nextProps.rate
+      	   });
+    	}
```

Comment:
These 5 lines contain a mix of spaces and tabs. Please replace with spaces only (like the rest of the file)